packages/cozy-client/src/store/queries.js

```diff
@@ -310,12 +311,12 @@ const getSelectorFilterFn = queryDefinition => {
 /**
  * Execute the given query against the document state.
  *
- * @param {import('../types').DocumentsStateSlice} state - The documents state
+ * @param {import('../types').CozyStore} store - The cozy store
  * @param {QueryDefinition} queryDefinition - The query definition to execute
  * @returns {import("../types").QueryStateData} - The returned documents from the query
  */
-export const executeQueryFromState = (state, queryDefinition) => {
-  const documents = getCollectionFromSlice(state, queryDefinition.doctype)
+export const executeQueryFromState = (store, queryDefinition) => {
```

Comment:
nit: Here, naming makes it very difficult to understand what's happening:
- In CozyClient we call `executeQueryFromState(this.store.getState(), ...`
  - Due to the `getState()` method we expect to manipulate a `state`
- But in the method definition, we expect a `store` parameter, so we are not talking about `state` anymore
- But then we call `getCollectionFromState` with `state` in its name and use `state` everywhere in its declaration

Reply:
I understand, I went back to this naming.